packages/Search/src/details/DTK_DetailsUtils.hpp

```diff
@@ -147,13 +148,39 @@ void iota( Kokkos::View<T, P...> const &v,
         std::is_same<ValueType, typename Kokkos::ViewTraits<
                                     T, P...>::non_const_value_type>::value,
         "iota requires a View with non-const value type" );
-    auto const n = v.span();
+    auto const n = v.extent( 0 );
     Kokkos::parallel_for(
         "iota", Kokkos::RangePolicy<ExecutionSpace>( 0, n ),
         KOKKOS_LAMBDA( int i ) { v( i ) = value + (ValueType)i; } );
     Kokkos::fence();
 }
 
+/** \brief Returns the smallest and the greatest element in the view
+ *
+ *  \param[in] v Input view
+ *
+ *  Returns a pair with the smallest value in the view as the first
+ *  element and the greatest as the second.
+ */
+template <typename ViewType>
+std::pair<typename ViewType::non_const_value_type,
+          typename ViewType::non_const_value_type>
+minmax( ViewType const &v )
+{
+    static_assert( ViewType::rank == 1, "minmax requires a View of rank 1" );
```

Comment:
Why make this restriction? It makes sense to run it on all views.

Reply:
1/ I don't think you can use Kokkos sort functionality on views that are not rank 1
2/ Give me a use case where you would be looking for minmax on a multi-dimensional array

Reply:
1. I see this minmax independent of `sort`.
2. Maybe choosing a pivot in dense factorization... Not sure.

Reply:
Well for now `minMax()` will only work with views of rank 1.  We may change the behavior later if needed.